modules/sc-mesh-secure-deployment/src/nats/src/cbma_adaptation.py

```diff
@@ -540,7 +558,7 @@ def __setup_radios(self) -> bool:
 
         return True
 
-    def setup_cbma(self) -> None:
+    def setup_cbma(self) -> bool:
```

Comment:
It doesn't look like this function can return `False` :heavy_check_mark:

Reply:
yes, but there is "return true" at the end and code editors are complaining "-> None vs return true".   Fail case handling needs to added.